src/Components/IngredientDetails/IngredientDetails.css

```diff
@@ -1,81 +1,188 @@
-.details-wrapper {
-  /* border: 2px solid purple; */
-  height: 85vh;
-  display: flex;
-  align-items: center;  
-  justify-content: center; 
-  font-family: 'HyliaSerifBeta-Regular';
-  color: white;
-}
+@media screen and (min-width: 950px) {
 
-.details-container {
-  background-color: rgb(0, 0, 0, .7);
-  height: 51vh;
-  width: 55%;
-  border-radius: 2%;
-  padding-bottom: .2em;
-  box-shadow: 0px 0.2em 0.5em #536f50;
-  /* box-shadow: 0px 0.2em 0.5em #7EF0F4; */
+  .details-wrapper {
+    /* border: 2px solid purple; */
+    height: 85vh;
+    display: flex;
+    align-items: center;  
+    justify-content: center; 
+    font-family: 'HyliaSerifBeta-Regular';
+    color: white;
+  }
+  
+  .details-container {
+    background-color: rgb(0, 0, 0, .7);
+    height: 51vh;
+    width: 65%;
+    border-radius: 2%;
+    padding-bottom: .2em;
+    box-shadow: 0px 0.2em 0.5em #536f50;
+    /* box-shadow: 0px 0.2em 0.5em #7EF0F4; */
+  }
+  
+  .ingredient-info {
+    display: flex;
+    height: 45vh;
+    /* border: 2px solid yellow; */
+  }
+  
+  .img-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    /* border: 2px solid orange; */
+    width: 40%;
+    padding: .6em;
+    /* border: 2px solid pink; */
+  }
+  
+  .details-img {
+    width: 75%;
+    /* height: 30vh; */
+    border-radius: 5%;
+    box-shadow: 0px 0.2em 0.5em #7EF0F4;
+    /* box-shadow: 0px 0.2em 0.5em #536f50; */
+  }
+  
+  .details-info {
+    text-align: center;
+    /* border: 2px solid red; */
+    width: 60%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    margin-right: .8em;
+    padding: .6em;
+    font-size: 1.125em;
+  }
+  
+  .button-container {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    /* border: 2px solid pink; */
+  }
+  
+  .link {
+    width: 15%;
+  }
+  
+  .back-btn {
```

Comment:
Adding a _border: none_  property and a little but of border-radius on this button might make this button look more modern instead of the generic border, but otherwise it the whole component looks good